helm-chart/templates/app.yaml

```diff
@@ -0,0 +1,105 @@
+---
+apiVersion: workload.crossplane.io/v1alpha1
+kind: KubernetesApplication
+metadata:
+  name: {{ .Release.Name }}-app
+  labels:
+    app: {{ .Release.Name }}
```

Comment:
Is .Release.Name rendering as expected? With helm2 I was observing that we didn't have access to .Release.Name in the same way as usual.

Reply:
Yes, it's been working for me.